lib/console/graphql/deployments/backup.ex

```diff
@@ -142,6 +144,20 @@ defmodule Console.GraphQl.Deployments.Backup do
       resolve &Deployments.resolve_cluster_backup/2
     end
 
+    connection field :cluster_backups, node_type: :cluster_backup do
+      middleware Authenticated
+      arg :cluster_id, non_null(:id)
```

Comment:
Will there be global list of backups (for all clusters)? I think I saw one in early designs, but I am not sure.

Reply:
I can make one but is there a need for it?  I'm actually not sure if it makes a lot of sense (could be super noisy)

Reply:
Yep, there could be a lot of backups if it will be configured for multiple clusters. Regarding clusters, how should we find list of clusters with backups enabled? One way is using object stores list and mapping them to clusters, would that be okay?

Reply:
Yep, makes sense.